### PR TITLE
Clarity updates

### PR DESCRIPTION
**Pull request by lriggs**
Some based on feedback from Arthur.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,35 +2,59 @@
...
 # What is a Connector?
 
 A connector is a set of files that describe:
 
 - UI elements needed to collect user input for creating a connection to a data source
 - Any dialect or customizations needed for the connection
+- How to connect using the ODBC or JDBC driver
 
-And include:
+A connector can most of the same features that any built-in Tableau connector supports, including publishing to a server if the server has the connector, creating extracts, data sources, vizzes, and so on.
```
Comment (sidwray) [inline docs/index.md]: Small typo: Should be "A connector has..."

File: docs/example.md [hunk 2]
```diff
@@ -93,7 +95,7 @@ You may also set any other connection string options that you would like to pass
 
 ## ![5]({{ site.baseurl }}/assets/pce-5.png) Connection Properties
 
-This script is needed only if you're using a JDBC driver.
+(Optional) This script is needed only if you're using a JDBC driver. The example below if for a connector to Amazon Athena.
```
Comment (sidwray) [inline docs/example.md]: Small typo: should be "The example below is... "

File: docs/index.md [hunk 3]
```diff
@@ -2,35 +2,59 @@
 title: Tableau Connector SDK
 ---
 
-With the Tableau Connector SDK, you can add a new connector that you can use to visualize your data from any database through an ODBC or JDBC driver.
-When you create a connector, you can add customizations to the connector, use the connectivity test harness to validate the connector behavior during the development process, and then package and distribute the connector to users.
-This document describes the files that make up a connector.
+With the Tableau Connector SDK, you can create a new connector that you can use to visualize your data from any database through an ODBC or JDBC driver.
+You can customize connector behavior, finely tune SQL generation, use the connectivity test harness to validate the connector behavior during the development process, and then package and distribute the connector to users.
```
Comment (lukewrites) [inline docs/index.md]:
```suggestion
You can customize connector behavior, fine tune SQL generation, use the connectivity test harness to validate the connector behavior during the development process, and then package and distribute the connector to users.
```

Comment (lukewrites) [reply docs/index.md]: nit 😄

File: docs/index.md [hunk 4]
```diff
@@ -2,35 +2,59 @@
...
+This document provides and overview of what a connector is and how it is created.
```
Comment (pvanderknyff) [inline docs/index.md]: Small typo
```suggestion
This document provides an overview of what a connector is and how it is created.
```

**Pull request merged**